0.59.md

```diff
@@ -0,0 +1,348 @@
+# 0.59
+
+Welcome to the February 2019 release of React Native! This is another big release with 644 commits by 88 contributors. Contributions also come in other forms, so *thank you* for maintaining issues, fostering communities, and teaching people about React Native. This month brings a number of highly anticipated changes, and we hope you enjoy them.
```

Comment:
Can we remove the "February" from here? As far as I know we are not back to a monthly release cycle but having that mentioned seem to suggest so.

Also, as we did for the other versions, can you please add the link to the diff with the commits & to the issue about the RC?